src/homepage/HomePage.js

```diff
@@ -5,12 +5,12 @@ import theme from '../theme/theme'
 import { colors } from '../theme/colors'
 import { mediaQueries } from '../theme/mediaQueries'
 import { Banner } from '../components/banner'
-import bannerBackground from './bannerBackground.jpg'
 import VolunteerBoxInfo from './VolunteerBoxInfo'
 import CampaignContainer from './CampaignContainer'
 import LatestNewsContainer from './LatestNewsContainer'
 import Announcements from './Announcements'
 import Donate from './Donate'
+import NewBackground from './NewBackground.jpg'
```

Comment:
Can we keep this as `bannerBackground.jpg` and just replace it with the new image? Else we have images that aren't imported + we don't have the history of the swap.